spec/features/content_pages_spec.rb

```diff
@@ -87,4 +88,38 @@ def content_urls
       end
     end
   end
+
+  content_urls.slice(0, 1).each do |first_url|
```

Comment:
`content_urls.first.tap do |first_url|` could be easier to follow here?

Reply:
:+1: